airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/connector_metrics.py

```diff
@@ -59,7 +59,7 @@ def _convert_json_to_metrics_dict(jsonl_string: str) -> dict:
 
 @asset(required_resource_keys={"latest_metrics_gcs_blob"}, group_name=GROUP_NAME)
 @sentry.instrument_asset_op
-def latest_connnector_metrics(context: OpExecutionContext) -> dict:
+def latest_connector_metrics(context: OpExecutionContext) -> dict:
```

Comment:
fixing typo

Reply:
Oh man, who would have been so careless....

![a-close-up-of-a-stuffed-monkey-wearing-a-green-shirt-and-blue-jeans](https://github.com/user-attachments/assets/f6e41de8-c2d3-46d7-9c60-190d8f1616f9)